BUILD.bazel

```diff
@@ -38,9 +38,14 @@ filegroup(
         "devtools/shared/**",
         "shells/**/*.js",
         "modalities/**/*",
+        "concrete-storage/*",
```

Comment:
This is fine to me, maybe @shans would want it still under `src/`? Thought worth checking.

Reply:
Can't go under `src` because it owns it's own `node_modules`, IIUC.

Reply:
best to check with Cameron as to where it should go, given that (I think) he's owning a repo reorg effort.

Reply:
If it has its own node_modules folder, then I think it makes sense as a sibling.

Maybe it should move to the `js` root folder which we're planning on adding? Then we'd have:
```
js/concrete-storage
js/runtime
etc.
```

Reply:
Same problem as `shells` iiuc, which is to say `js` is formerly known as `src` and `src` is mirrored into `build`, which is not what we want for `concrete-storage`.

Reply:
Hmm, ok. It's probably fine for now, but part of me feels like we have too many top-level folders, and should tuck them away somewhere. It might be nice to have a folder for all of our non-sigh npm builds.

A problem for another day...